uberfire-project/uberfire-project-backend/src/main/java/org/guvnor/common/services/project/backend/server/ModuleRepositoryResolverImpl.java

```diff
@@ -171,6 +183,31 @@ public ModuleRepositoryResolverImpl(final @Named("ioStrategy") IOService ioServi
         return repositories;
     }
 
+    private RepositorySystemSession newSession(RepositorySystem system) {
+        ArtifactRepositoryPreference artifactRepositoryPreference = new ArtifactRepositoryPreference();
+        artifactRepositoryPreference.defaultValue(artifactRepositoryPreference);
+        DefaultRepositorySystemSession session = MavenRepositorySystemUtils.newSession();
+        String global = artifactRepositoryPreference.getGlobalM2RepoDir();
+        if (global == null) {
+            global = "repositories" + File.separator + "kie" + File.separator + "global";
```

Comment:
not much happy with the `default` global has been spread around. could you please centralize it in just one place?

Reply:
Fixed on both branches